examples/grpc/src/main/java/example/armeria/grpc/HelloServiceImpl.java

```diff
@@ -20,7 +21,11 @@
      */
     @Override
     public void hello(HelloRequest request, StreamObserver<HelloReply> responseObserver) {
-        responseObserver.onNext(buildReply(toMessage(request.getName())));
+        if (request.getName().isEmpty()) {
+            responseObserver.onError(Status.FAILED_PRECONDITION.withDescription("Name cannot be empty").asException());
+        } else {
+            responseObserver.onNext(buildReply(toMessage(request.getName())));
+        }
         responseObserver.onCompleted();
```

Comment:
`onCompleted()` might not be expected after `onError()`.
```suggestion
        if (request.getName().isEmpty()) {
            responseObserver.onError(Status.FAILED_PRECONDITION.withDescription("Name cannot be empty").asException());
        } else {
            responseObserver.onNext(buildReply(toMessage(request.getName())));
            responseObserver.onCompleted();
        }

```